src/network/http_server.rs

```diff
@@ -471,14 +492,16 @@ pub enum IdRegisterEventType {
 
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct SignerEventBody {
-    pub key: Vec<u8>,
+    #[serde(with = "serdebase64")]
```

Comment:
based on hub code i think this should be hex

Reply:
Ok thanks - making the change right now